CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Added
 
 - Fast promotion tracks for the Fellowship ranks I-III ([polkadot-fellows/runtimes#356](https://github.com/polkadot-fellows/runtimes/pull/356)).
+- Migration to remove all but the 21 first elected Head Ambassador members from the Program ([polkadot-fellows/runtimes#422](https://github.com/polkadot-fellows/runtimes/pull/422)).
```

Comment:
Why? Because we have now an upper maximum? Would be nice to have somewhere documented the reason.

Reply:
Why not do this as public referendum?

Reply:
We discussed with a few HAs and thought this would be better. Doing public referenda would put us back into the same situation, where a number of simultaneous referenda pass and remove more than necessary (which, TBH, is also fine with me). This solution just reduces the number of referenda and makes the selection criteria clear: first 21.

Reply:
Can we not remove them in a batch? But if you discussed this, fine.

Reply:
Yeah but it's the same thing, people can put up competing batches and we don't have a system to say "just pick one of these referenda". I am also fine with either solution -- the migration to keep the first 21 or allow free-for-all removal referenda. Either way new people won't be able to join until it's below 21.

Reply:
Yea Free-for-all-removals could also work, but then they would probably remove too many.  
Without the migration we are in a constant race-condition with other proposals.